packages/build-tools/src/steps/functions/saveCache.ts

```diff
@@ -76,29 +73,39 @@ export function createSaveCacheFunction(): BuildFunction {
 
 export async function uploadCacheAsync({
   logger,
-  jobRunId,
+  buildId,
```

Comment:
Replacing `jobRunId` with `buildId` makes it weird for job runs? For single arguments like these I've been using `taskId` (from lack of better options). Maybe `jobId` would be good too.

We can also consider passing in a `ctx: OneOfBuildContexts` and let `uploadCacheAsync` figure out:
- whether we're in a build or a job run
- robot access token
- the path to use
etc.